src/storage/copier/var_list_column_chunk.cpp

```diff
@@ -11,6 +11,16 @@ void VarListDataColumnChunk::reset() {
     numValuesInDataChunk = 0;
 }
 
+void VarListDataColumnChunk::resize(uint64_t numValues) {
+    if (numValues <= capacityInDataChunk) {
+        return;
+    }
+    while (capacityInDataChunk < numValues) {
+        capacityInDataChunk *= 2;
```

Comment:
Convert 2 into a constant? like `VAR_LIST_RESIZE_RATIO`. Same for `ListAuxiliaryBuffer::resize`. We can use the same constant.